tools/generate_func.py

```diff
@@ -257,7 +257,10 @@ def cleanup(name):
                 else:
                     print('int %s=-2**31' % var, end=' ')   # TA_INTEGER_DEFAULT
             elif arg.startswith('TA_MAType'):
-                print('int %s=0' % var, end=' ')            # TA_MAType_SMA
+                if 'matype' in defaults:
+                    print('int %s=%s' % (var, defaults['matype']), end=' ')
```

Comment:
I'm not a python developer, so... Could you edit my PR?

Reply:
Or you may create a new one from a scratch by yourself - I'll delete this one.

Reply:
On a second thougth ...
```
if 'matype' in defaults:
                    print('int %s=%s' % (var, defaults['matype']), end=' ')
```
is not equal to just `print('int %s=0' % var, end=' ')`. Bcs it'll do nothing in case TA_MAType argument exists but have no any default value. Although I think there is no such cases in the sourcecode.